### PR TITLE
Telepresence 2.8.2 (new formula)

### PR DESCRIPTION
**Pull request by marknefedov**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/telepresence.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "build-output/bin/telepresence"
+  end
+
+  test do
+    system "#{bin}/telepresence", "status"
```
Comment (SMillerDev) [inline Formula/telepresence.rb:22]: Can we check some output here?

File: Formula/telepresence.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "build-output/bin/telepresence"
```
Comment (SMillerDev) [inline Formula/telepresence.rb:18]: no `make install`?

File: Formula/telepresence.rb (hunk 1 above)
Comment (markusgod) [reply Formula/telepresence.rb:22]:
```
❯ telepresence status
Root Daemon: Not running
User Daemon: Not running
```
Any tips how should I check multiline output, I'm barely worked with Ruby.

File: Formula/telepresence.rb (hunk 2 above)
Comment (markusgod) [reply Formula/telepresence.rb:18]: There is one, that I didn't notice, I'll update it.

**Comment by markusgod** [comment]
Git exploded. nice...
How can I re-open pull-request? I didn't mean close it, and have no idea how I did.

**Comment by SMillerDev** [comment]
You removed all commits, which closed the PR.

**Pull request closed without merging**